### ISSUE TITLE
Artikkel: Dotted underline på kategori og skribent

### ISSUE THREAD

**Issue by tonenordbo** [UX-Norge/ux-norge#115]

**Comment by ksannaes** [comment]
Det er den samme klassen (Tailwinds decoration-dotted), så det skal være likt. Men kanskje det er forskjell på line-height på de to tekstene? Det finnes en annen klasse som kan brukes, som heter decoration-dashed. Kan den være et alternativ (se skjermbilde fra dokumentasjonen)?

![Image](https://github.com/user-attachments/assets/833a0238-790e-48a5-9522-aed1563e4b47)

Alternativt kan vi bruke border-bottom, der vi har litt mer kontroll på størrelsene. Da kommer den muligens enda litt lenger ned.

**Comment by tonenordbo** [comment]
Ja, kan bruke dashed. Men sjekk gjerne forskjellen på de to da, er et poeng at de ser like ut.

**Comment by ksannaes** [comment]
Jeg tester litt nå for å finne ut av forskjellene. Det ser litt annerledes ut så langt i mine tester enn i skjermbildet du sendte, hvilken nettleser var det i?

Jeg oppdaget at hos meg var den ene underlinjen sort, så det har jeg fikset. Men i Chrome er det veldig liten avstand mellom tekst og strek. Det får jeg ikke gjort noe med med understreking, men kan styre det noe med border, om du vil at jeg skal velge det i stedet. Da blir det lenger avstand overalt, antakeligvis.

**Comment by ksannaes** [comment]
Jeg har flyttet litt på html-elementer, sånn at rekkefølgen på elementer og klasser er lik for kategori og forfatternavn. Hos meg ser det ut til at de to er like hos meg nå. Det er jo noen forskjeller mellom nettlesere, men da likt for begge feltene. Se skjermbilde som viser Firefox til venstre og Safari til høyre.

![Image](https://github.com/user-attachments/assets/e02e8bf8-99a0-40e3-b4b6-eaf94e2c7ea8)
